{"name":"To verify user not able to SignIn with invalid credentials","status":"failed","statusDetails":{"known":false,"muted":false,"flaky":false,"message":"Try to login with invalid email/phonenumber expected [false] but found [true]","trace":"java.lang.AssertionError: Try to login with invalid email/phonenumber expected [false] but found [true]\r\n\tat org.testng.Assert.fail(Assert.java:99)\r\n\tat org.testng.Assert.failNotEquals(Assert.java:1037)\r\n\tat org.testng.Assert.assertFalse(Assert.java:67)\r\n\tat pageTestCases.LoginTest.invalidSignIn(LoginTest.java:91)\r\n\tat java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)\r\n\tat java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:64)\r\n\tat java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)\r\n\tat java.base/java.lang.reflect.Method.invoke(Method.java:564)\r\n\tat org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:132)\r\n\tat org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:599)\r\n\tat org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:174)\r\n\tat org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)\r\n\tat org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:822)\r\n\tat org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:147)\r\n\tat org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)\r\n\tat org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)\r\n\tat java.base/java.util.ArrayList.forEach(ArrayList.java:1511)\r\n\tat org.testng.TestRunner.privateRun(TestRunner.java:764)\r\n\tat org.testng.TestRunner.run(TestRunner.java:585)\r\n\tat org.testng.SuiteRunner.runTest(SuiteRunner.java:384)\r\n\tat org.testng.SuiteRunner.runSequentially(SuiteRunner.java:378)\r\n\tat org.testng.SuiteRunner.privateRun(SuiteRunner.java:337)\r\n\tat org.testng.SuiteRunner.run(SuiteRunner.java:286)\r\n\tat org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)\r\n\tat org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)\r\n\tat org.testng.TestNG.runSuitesSequentially(TestNG.java:1218)\r\n\tat org.testng.TestNG.runSuitesLocally(TestNG.java:1140)\r\n\tat org.testng.TestNG.runSuites(TestNG.java:1069)\r\n\tat org.testng.TestNG.run(TestNG.java:1037)\r\n\tat org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)\r\n\tat org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)\r\n\tat org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)\r\n"},"stage":"finished","description":"Test Case Description:To verify user not able to SignIn with Invalid Credentials","steps":[{"name":"Method to open Signin form","status":"passed","stage":"finished","steps":[],"attachments":[],"parameters":[],"start":1620968237591,"stop":1620968237597},{"name":"Method to enter email or mobile no","status":"passed","stage":"finished","steps":[],"attachments":[],"parameters":[],"start":1620968240796,"stop":1620968240796},{"name":"Method to click on Continue","status":"passed","stage":"finished","steps":[],"attachments":[],"parameters":[],"start":1620968240915,"stop":1620968240915},{"name":"Method to show invalid email response","status":"passed","stage":"finished","steps":[],"attachments":[],"parameters":[],"start":1620968241691,"stop":1620968241691}],"attachments":[{"name":"Page Screenshot","source":"2ae87a3a-f8aa-4ac3-8d18-85249fd854e9-attachment","type":"image/png"},{"name":"invalidSignInFailed and screenshot Taken!","source":"484e5b4d-1f45-421e-9a4f-01544f51be26-attachment","type":"text/plain"}],"parameters":[],"start":1620968232951,"stop":1620968241991,"uuid":"183c3bf0-aff1-46ec-9bfb-cb6580ce0fae","historyId":"af4eb947f79d6d11833e155ce3d35b98","fullName":"pageTestCases.LoginTest.invalidSignIn","labels":[{"name":"package","value":"pageTestCases.LoginTest"},{"name":"testClass","value":"pageTestCases.LoginTest"},{"name":"testMethod","value":"invalidSignIn"},{"name":"parentSuite","value":"Suite"},{"name":"suite","value":"Test"},{"name":"subSuite","value":"pageTestCases.LoginTest"},{"name":"host","value":"PC"},{"name":"thread","value":"8852@PC.main(1)"},{"name":"framework","value":"testng"},{"name":"language","value":"java"},{"name":"story","value":"Story Name: To check user not able to SignIn with invalid credentials"},{"name":"severity","value":"critical"}],"links":[]}